style.css

```diff
@@ -175,10 +145,81 @@ h2 a:visited {
     color: var(--blue);
 }
 
+.filter-question .btn {
+    border-radius: 0;
+}
+
+.filter-question .btn:first-child {
+    border-bottom-left-radius: 0.2em;
+    border-top-left-radius: 0.2em;
+}
+
+.filter-question .btn:nth-last-child(2) {
+    border-bottom-right-radius: 0.2em;
+    border-top-right-radius: 0.2em;
+}
+
```

Comment:
Previously this code was on line `135`. You moved it down, and now looks like changed (see picture)
<img width="697" alt="image" src="https://user-images.githubusercontent.com/11327761/201514834-727e9c76-0ce3-4634-9d01-386de8a025e5.png">

Please put it back to it;s place, so we don't see it at "files changed"